R/get_pro.R

```diff
@@ -92,7 +92,7 @@ get_pro_events <- function(urlname,
   api_path <- sprintf("pro/%s/events", urlname)
   res <- .fetch_results(api_path = api_path, verbose = verbose)
 
-  group <- lapply(res, function(x) x[[3]])
+  group <- lapply(res, function(x) x[["chapter"]])
```

Comment:
@Athanasiamo sometimes x is of length 2 only, when the event has no image (or the chapter had no image? I can't remember now)

Reply:
great! ys, definitely better to index by name!